packages/ui/src/molecules/ProductCard/ProductCard.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { render } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import ProductCard from './ProductCard'
+import ProductCardImage from './ProductCardImage'
+import ProductCardContent from './ProductCardContent'
+import ProductCardActions from './ProductCardActions'
+
+const TestProductCard = () => {
+  return (
+    <ProductCard>
+      <ProductCardImage>
+        <div>An image</div>
+      </ProductCardImage>
+      <ProductCardContent>
+        <h3>A title</h3>
+        <p>A little text here: The quick brown fox jumps over the lazy dog.</p>
+      </ProductCardContent>
+      <ProductCardActions>
+        <button>A button</button>
+      </ProductCardActions>
+    </ProductCard>
+  )
+}
+
+describe('ProductCard', () => {
+  let productCard: HTMLElement
+  let productCardImage: HTMLElement
+  let productCardContent: HTMLElement
+  let productCardActions: HTMLElement
+  let productCardContainer: HTMLElement
+
+  beforeEach(() => {
+    const { getByTestId, container } = render(<TestProductCard />)
+
+    productCardContainer = container
+    productCard = getByTestId('store-product-card')
+    productCardImage = getByTestId('store-product-card-image')
+    productCardContent = getByTestId('store-product-card-content')
+    productCardActions = getByTestId('store-product-card-actions')
+  })
+
+  describe('Data attributes', () => {
+    it('`Card` component should have `data-store-product-card` attribute', () => {
+      expect(productCard).toHaveAttribute('data-store-product-card')
+    })
+
+    it('`CardContent` component should have `data-product-card-content` attribute', () => {
```

Comment:
If CardContent is always a section, should have a test for this.

Reply:
Okay! Doneee [here](https://github.com/vtex/faststore/pull/1272/commits/e7c8cb0b7c4280605cb04d3aacddd1e2a0245fcf)